### PR TITLE
Implement static elements for the screen 1

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1076
[Problem] - Page elements doesn't exists
[Solution] - Make static elements design

### CODE REVIEW

File: examples/mobile/ScheduleApp/index.html [hunk 1]
```diff
@@ -26,6 +26,106 @@
...
+					<div class="ui-li-action action-container">
+						<span class="date-text">Today</span>
+						<select class="ui-on-off-switch">
+							<option value="off"></option>
+							<option value="on" selected></option>
+						</select>
+					</div>
+				</div>
+				<div class="content-footer">
+					Cool/ 18°C/ High
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html]: I suggest that "Cool / 18°C / High".

File: examples/mobile/ScheduleApp/index.html [hunk 2]
```diff
@@ -26,6 +26,106 @@
...
+					<div class="hour-container">
+						<span class="ui-text hour-value">
+							2:30
+						</span>
+						<span class="ui-text hour-signature">
+							PM
+						</span>
+					</div>
+					<div class="ui-li-action action-container">
+						<span class="date-text">Today</span>
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html:43]: Format the 'span' the same.
The top or bottom form is unified.

File: examples/mobile/ScheduleApp/index.html [hunk 3]
```diff
@@ -26,6 +26,106 @@
...
+					<div class="hour-container">
+						<span class="ui-text hour-value">
+							6:35
+						</span>
+						<span class="ui-text hour-signature">
+							AM
+						</span>
+					</div>
+					<div class="ui-li-action">
+						<span class="date-text">S M T W T F S</span>
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html:68]: ditto.

File: examples/mobile/ScheduleApp/index.html [hunk 4]
```diff
@@ -26,6 +26,106 @@
...
+					<div class="hour-container">
+						<span class="ui-text hour-value">
+							7:00
+						</span>
+						<span class="ui-text hour-signature">
+							AM
+						</span>
+					</div>
+					<div class="ui-li-action">
+						<span class="date-text">S M T W T F S</span>
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html:93]: ditto.

File: examples/mobile/ScheduleApp/index.html [hunk 5]
```diff
@@ -26,6 +26,106 @@
...
+						<select class="ui-on-off-switch">
+							<option value="off"></option>
+							<option value="on" selected></option>
+						</select>
+					</div>
+				</div>
+				<div class="content-footer">
+					Cool/ 18°C/ High
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html]: ditto.

File: examples/mobile/ScheduleApp/index.html [hunk 6]
```diff
@@ -26,6 +26,106 @@
...
+					<div class="hour-container">
+						<span class="ui-text hour-value">
+							7:06
+						</span>
+						<span class="ui-text hour-signature">
+							AM
+						</span>
+					</div>
+					<div class="ui-li-action">
+						<span class="date-text">S M T W T F S</span>
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html:118]: ditto.

File: examples/mobile/ScheduleApp/index.html [hunk 7]
```diff
@@ -26,6 +26,106 @@
...
+						<select class="ui-on-off-switch">
+							<option value="off"></option>
+							<option value="on" selected></option>
+						</select>
+					</div>
+				</div>
+				<div class="content-footer">
+					Cool/ 18°C/ High
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/index.html]: ditto.

File: examples/mobile/ScheduleApp/css/style.css [hunk 8]
```diff
@@ -1,3 +1,46 @@
...
+}
+
+.action-container {
+	display: flex;
+}
+
+.content-footer {
+	color: #909090;
+}
+.date-text {
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/css/style.css:40]: need to enter.

**Review by DohyungLim — COMMENTED** [review]
8 comment.

File: examples/mobile/ScheduleApp/index.html (hunk 3 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html:68]: Done

File: examples/mobile/ScheduleApp/index.html (hunk 4 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html:93]: Done

File: examples/mobile/ScheduleApp/index.html (hunk 5 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html]: done

File: examples/mobile/ScheduleApp/index.html (hunk 7 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html]: Done

File: examples/mobile/ScheduleApp/index.html (hunk 6 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html:118]: Done

File: examples/mobile/ScheduleApp/css/style.css (hunk 8 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css:40]: Done

File: examples/mobile/ScheduleApp/css/style.css [hunk 9]
```diff
@@ -1,3 +1,47 @@
...
+	align-items: baseline;
+}
+
+.hour-value {
+	font-size: 38px;
+	font-family: 'Roboto-Light';
+}
+
+.hour-signature {
+	margin-left: 4px;
```
Comment (lslachciak) [inline examples/mobile/ScheduleApp/css/style.css]: whitespaces on the end of line
image

**Review by lslachciak — APPROVED** [review]
OK, 
please merge after fixing whitespace issue

File: examples/mobile/ScheduleApp/css/style.css (hunk 9 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/css/style.css]: Done!

**Pull request merged**